### ISSUE TITLE
GKE scalability jobs failing due to bad JSON config

### ISSUE THREAD

**Issue by shyamjvs** [kubernetes/test-infra#9165]
We're seeing failures like:

```
Something went wrong: error creating deployer: --gke-shape must be valid JSON, unmarshal error: invalid character '\'' looking for beginning of value, JSON: "'{\"default\":{\"Nodes\":4999,\"MachineType\":\"g1-small\"},\"heapster-pool\":{\"Nodes\":1,\"MachineType\":\"n1-standard-16\"}}'"
```

Affected jobs:

- https://k8s-testgrid.appspot.com/sig-scalability-gke#gke-scale-correctness
- https://k8s-testgrid.appspot.com/sig-scalability-gke#gke-large-performance
- https://k8s-testgrid.appspot.com/sig-scalability-gke#gke-large-performance-regional

/assign @krzyzacy @BenTheElder 
Could one of you PTAL? Thanks.

cc @kubernetes/sig-scalability-misc

**Comment by BenTheElder** [comment]
This flag (`--gke-shape`) is clearly not escaped properly.
https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-scalability/sig-scalability-config.yaml
Probably incorrectly quoted after moving the configs around.

**Comment by shyamjvs** [comment]
Right. I tried fixing it (unsuccessfully) earlier in - https://github.com/kubernetes/test-infra/commit/d5bc6e92aaccf70ce3e945d1afdf399b8e7233d7#diff-1664f4f6e4712f8d89861b94e122dd83
Do you know what's the right way here?

**Comment by shyamjvs** [comment]
I tried following this suggestion - https://stackoverflow.com/questions/9532840/embedding-json-data-into-yaml-file - but that doesn't seem to work.

**Comment by BenTheElder** [comment]
Thanks @krzyzacy, yeah we should quote the entire argument instead when writing this in yaml

**Comment by krzyzacy** [comment]
(fixed by https://github.com/kubernetes/test-infra/pull/9168 ^^)
